include/OnnxMlirCompiler.h

```diff
@@ -91,8 +91,10 @@ ONNX_MLIR_EXPORT int64_t omSetCompilerOptionsFromArgsAndEnv(
  *  omSetCompilerOptionsFromEnvAndArgs. Or overwrite the current value
  *  by calling this function after one of the above 3 setter functions.
  *  @param kind Describe which option kind is being set.
- *  @param val Value of the option being set. Null pointer reset the
- *  option.
+ *  @param val Value of the option being set. Empty string reset the
+ *  option. Setting TargetAccel is different. When passing a valid
+ *  accelerator, it is added to the list of target accelerators unless
+ *  val="RESET", in which case the list is cleared.
```

Comment:
Given that the maccel command line option accept "NONE" to indicate there are no accelerators I wonder whether would make sense to also use "NONE" instead of "RESET" here. What do you think?

Reply:
That make sense, but there is now a slight inconstancy: if on the command line, a user add `--maccel=NNPA --maccel=NONE` the LLVM mechanics does not know that `NONE` invalidates the other ones. And what if a user say `--maccel=NNPA --maccel=NONE --maccel=NNPA2`? Does it process them linearly, and thus have only `NNPA2`, or the `NONE` apply for all?

That is why I preferred avoiding this issue altogether by adding a distinct name, which is only used by the `setCompilerOption`. The reason it is important there to reset, is that a program that compile itself graphs may want to generate two versions, one with and one without accel. So it has to be able to reset as well as add.

One way to avoid this all together is to treat the string passed to  `setCompilerOption` as a set, namely it always reset the previous set and can add multiple, e.g. comma separated accelerators. That would probably be the cleanest as it would match the other options that simply erase the previous option and reset it with the new ones... Would you be ok with comma separated string for describing a set?

Reply:
Ok, after investigation, the command line accepts multiple `--maccel=xxx` and add them all together. It does not recognize `--maccel=xxx1,xxx2`. So to be as similar as possible to the command line option, we are keeping the current approach of a special keyword, `RESET` to clear the set when passing option using the `setCompilerOption` approach.